docs/shared/deprecated/legacy-cache.md

```diff
@@ -0,0 +1,22 @@
+# Legacy Cache
+
+## tasksRunnerOptions
+
+As of Nx 20, the `tasksRunnerOptions` property in `nx.json` is deprecated. This property was used to register custom task runners. `tasksRunnerOptions` and custom task runners only work with the legacy file system cache.
+
+## NX_REJECT_UNKNOWN_LOCAL_CACHE
+
+The `NX_REJECT_UNKNOWN_LOCAL_CACHE` environment variable only works with the legacy file system local cache.
```

Comment:
These 2 seem too far up. I think these should be below the section below?

Reply:
Those are the reason people are landing on the page.  All the deprecated pages are organized like this:

1. This is what features X did.
2. This is when it was deprecated and removed
3. This is what you should do to move forward